packages/SwingSet/test/timer/test-timer.js

```diff
@@ -72,4 +72,48 @@ test('timer vat', async t => {
 
   const cd9 = await run('banana', [10n]);
   t.deepEqual(parse(cd9.body), 'bad setWakeup() handler');
+
+  // start a repeater that should first fire at now+delay+interval, so
+  // 7+20+10=27,37,47,57,..  TODO: poll at 25,35,40
```

Comment:
Why `TODO`?   The test does poll() 25,35,40.

Reply:
you're right, that's stale, I'll remove it